lib/tasks/mastodon.rake

```diff
@@ -571,6 +572,7 @@ namespace :mastodon do
   namespace :webpush do
     desc 'Generate VAPID key'
     task :generate_vapid_key do
+      # Uses Webpush class, Webpush::Legacy does the same thing.
```

Comment:
I think this can be removed, `Webpush::Legacy` (as in, in the gem maintained by Mastodon) does not cover this at all.